src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Assign Emission Admin - Ethereum, Arbitirum and Optimism"
+author: "karpatkey-TokenLogic & ACI"
+---
+
+## Simple Summary
+
+This enables various teams to distribute rewards across Aave v3 Ethereuim.
+
+## Motivation
+
+This AIP integrates four separate ARFC that each propose granting a team the privileges to distribute there choose asset across an Aave v3 deployment. The below summaries each iniative:
+
+- ETHx & SD rewards by Stader Labs
+- osETH & SWISE by Stakewise DAO
+- OP reward via an Aave Community SAFE
+- ARB rewards via Gauntlet
+
+The two LST providers are intending on using rewards to bootstrap the growth and adoption of there resepective LST.
+
+The OP and ARB incentives are being distributed with the intent of migrating USDC.e to USDC on each respective network. These ARB and OP rewards were originally provide by the respective foundation and are not those belonging to the Aave DAO.
+
+## Specification
+
+The `EmissionManager`, a governance controlled function, assigns the `Emission_Admin` role for a specified token on the respective network.
```

Comment:
weird mix of writing: Now you have `EmissionManager`, `Emission_Admin` and later `EMISSION_MANAGER` ... better to just stay consistent (I'd suggest with the last one)